docs/changehistory/NextVersion.md

```diff
@@ -54,3 +54,7 @@ export class ExtensionUiItemsProvider implements UiItemsProvider {
 - If an application calls [UiFramework.setIModelConnection]($ui-framework) it will no longer need to explicitly call [SyncUiEventDispatcher.initializeConnectionEvents]($ui-framework) as `UiFramework.setIModelConnection` will call that method as it update the redux store.
 
 - The `version` prop passed to [FrameworkVersion]($ui-framework) component will update the [UiFramework.uiVersion] if necessary keeping the redux state matching the value defined by the prop.
+
+- The [ScheduleAnimationProvider]($ui-framework) is published for use by AppUi apps. Specifying this data provider to a [TimelineComponent]($ui-components) allows animation of the [RenderSchedule.Script]($common) if one exists for the view. A component that automatically detects a schedule script and attaches the data provider to its TimelineComponent can be found in the [DefaultViewOverlay]($ui-framework).
+
+- The [AnalysisAnimationProvider]($ui-framework) is published for use by AppUi apps. Specifying this data provider to a TimelineComponent allows animation of the information in the AnalysisDisplayProperties if the view's [DisplayStyleState]($frontend) contains one. A component that automatically detects analysis data and attaches the data provider to its TimelineComponent can be found in the [DefaultViewOverlay]($ui-framework).
```

Comment:
`/s/AnalysisDisplayProperties/AnalysisStyle`.
I have a PR in progress that will promote this class from alpha to public - I'll try to remember to update this to link to the docs once that's completed.

Reply:
I'm doing some other work on the TimelineComponent. I'll try to remember to check when I get ready to PR it.

Reply:
There's no such thing as AnalysisDisplayProperties. It doesn't matter - I will change it in my PR after I merge yours.